src/style.css

```
@@ -1 +1,46 @@
-/* add styles here */
+
+body {
+margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

```suggestion
.stars--0 .stars__star:nth-child(-n + 0),
.stars--5 .stars__star:nth-child(-n + 5),
... {
```
Consider using commas for grouping selectors